android/modules/ui/src/java/ti/modules/titanium/ui/widget/tableview/TableViewHolder.java

```diff
@@ -209,13 +224,13 @@ public void bind(final TableViewRowProxy proxy, final boolean selected)
 			// Handle row left and right images.
 			if (properties.containsKeyAndNotNull(TiC.PROPERTY_LEFT_IMAGE)) {
 				final String url = properties.getString(TiC.PROPERTY_LEFT_IMAGE);
-				final Drawable drawable = fileHelper.loadDrawable(url, false);
+				final Drawable drawable = TiUIHelper.getResourceDrawable(url);
 				this.leftImage.setImageDrawable(drawable);
 				this.leftImage.setVisibility(View.VISIBLE);
 			}
 			if (properties.containsKeyAndNotNull(TiC.PROPERTY_RIGHT_IMAGE)) {
 				final String url = properties.getString(TiC.PROPERTY_RIGHT_IMAGE);
-				final Drawable drawable = fileHelper.loadDrawable(url, false);
+				final Drawable drawable = TiUIHelper.getResourceDrawable(url);
```

Comment:
This is failing to display images for me now... and I don't understand why. 😕 

I think it's loading the image fine because there are no warnings in the log. Assigning the "leftImage" make the "title" move slightly to the right. Maybe the issue is the ImageView isn't resizing itself correctly or there is something wrong with the loaded drawable? I dunno.